packages/flutter_tools/test/integration.shard/analyze_size_test.dart

```diff
@@ -187,6 +187,9 @@ void main() {
       '--release',
     ], workingDirectory: fileSystem.path.join(getFlutterRoot(), 'examples', 'hello_world'));
 
+    printOnFailure('stdout:\n${result.stdout}');
```

Comment:
Can you pull the list literal we're passing to `processManager.run()` into a variable and print that. Same for `workingDirectory`, in case the problem lies there.

Reply:
Yep, good to go!